web-registry/src/components/molecules/AdditionalMetadata.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
```

Comment:
might want to name this `AdditionalProjectMetadata`

Reply:
done